features/support/webmock.rb

```diff
@@ -1,5 +1,7 @@
 require 'webmock/cucumber'
 require_relative './capybara_driver_helper'
+
 selenium_url = URI.parse ENV.fetch('SELENIUM_URL', 'http://localhost:4444/wd/hub')
 app_host_url = URI.parse Capybara.app_host
 WebMock.disable_net_connect!(allow_localhost: true, allow: [selenium_url.host, app_host_url.host, 'ondemand.saucelabs.com'])
+WebMock.disable_net_connect!(allow_localhost: true, allow: 'chromedriver.storage.googleapis.com')
```

Comment:
How about you add the chromedriver url in the line 6?
```suggestion
WebMock.disable_net_connect!(allow_localhost: true, allow: [selenium_url.host, app_host_url.host, 'ondemand.saucelabs.com', 'chromedriver.storage.googleapis.com'])
```